metrics/size.table/query.go

```diff
@@ -0,0 +1,55 @@
+package sizetable
+
+import (
+	"fmt"
+	"strings"
+)
+
+func TableSizeQuery(set map[string]string) (string, error) {
+	query := "SELECT table_schema AS db, table_name as tbl, COALESCE(data_length + index_length, 0) AS tbl_size_bytes FROM information_schema.TABLES"
+	where := " WHERE "
+	if include := set[OPT_INCLUDE]; include != "" {
+		var include_tables []string
+		if strings.Contains(set[OPT_INCLUDE], ",") {
+			include_tables = strings.Split(set[OPT_INCLUDE], ",")
+		} else {
+			include_tables = []string{set[OPT_INCLUDE]}
+		}
+		where = setWhere(include_tables, true, where)
+	} else {
+		where = where + "NOT "
+		var exclude_tables []string
+		if strings.Contains(set[OPT_EXCLUDE], ",") {
```

Comment:
More idiomatic
```go
excludeTables := strings.Split(set[OPT_EXCLUDE], ",")
if len(excludeTables) > 0 {
    setWhere(...)
}
```

Or more compactly:

```go
setWhere(strings.Split(set[OPT_EXCLUDE], ","), <other args>)
```
because a `for` over an empty list (in `setWhere()`) will simply do nothing.

Either way, the `else {` branch here isn't needed or the standard way of writing Go.

Reply:
I thought that if I try to split a string on something that it didn't contain would result in an error, but I realize now after trying it out in golang playground that it doesn't.